components/local_date_time/local_date_time.tsx

```diff
@@ -29,37 +28,40 @@ type Props = {
 }
 
 export default class LocalDateTime extends React.PureComponent<Props> {
-    public render() {
+    getFormattedTime = () => {
```

Comment:
Shall we define a return type here? easier to consume that way?

Reply:
Sure, now updated. PTAL.
It feels overkill to require typing for function that is to be consumed locally (by the component itself). I'd prefer inferred type in this case though.

Reply:
Personally prefer being explicit in return types for helper functions then it's clear on what's being returned. Fair point on keeping a balance between being extremely verbose vs not. I'm planning on putting out JS/TS conventions for our codebase soon, hopefully we can resolve the discussion there. If we choose not to be explicit we can always come back and remove these :).